### PR TITLE
Feature/upgrade 11 12

### PR DESCRIPTION
**Pull request by laurens-ritense**

### CODE REVIEW

File: slack/.gitignore
Comment (Klaas-Ritense) [inline slack/.gitignore:1]: This .gitignore file is not needed here. It should only be in the root of the repository

File: slack/.idea/.gitignore
Comment (Klaas-Ritense) [inline slack/.idea/.gitignore:1]: This .idea file and other, should not be committed

File: slack/.idea/gradle.xml
Comment (Klaas-Ritense) [inline slack/.idea/gradle.xml:1]: Could you remove all files in this /slack/.idea folder

File: slack/frontend/src/app/app.module.ts [hunk 1]
```diff
@@ -55,7 +53,6 @@ import {DecisionModule} from '@valtimo/decision';
 import {MilestoneModule} from '@valtimo/milestone';
 import {LoggerModule} from 'ngx-logger';
 import {FormManagementModule} from '@valtimo/form-management';
-import {FormLinkModule} from '@valtimo/form-link';
```
Comment (Klaas-Ritense) [inline slack/frontend/src/app/app.module.ts:58]: Its missing a ProcessLinkModule

File: slack/frontend/package.json [hunk 2]
```diff
@@ -18,100 +18,97 @@
...
-    "@angular/compiler": "^16.2.0",
-    "@angular/core": "^16.2.0",
-    "@angular/elements": "^16.2.0",
-    "@angular/forms": "^16.2.0",
-    "@angular/localize": "^16.2.0",
-    "@angular/platform-browser": "^16.2.0",
-    "@angular/platform-browser-dynamic": "^16.2.0",
-    "@angular/router": "^16.2.0",
-    "@mdi/font": "7.2.96",
+    "@angular/animations": "^17.2.2",
```
Comment (Klaas-Ritense) [inline slack/frontend/package.json:21]: There is another package.json which still needs the angular upgrade to 17

File: slack/backend/plugin/src/test/resources/config/application.yml
Comment (Klaas-Ritense) [inline slack/backend/plugin/src/test/resources/config/application.yml:1]: Can you also try running all integration tests

File: slack/frontend/package.json (hunk 2 above)
Comment (laurens-ritense) [reply slack/frontend/package.json:21]: Which?

File: slack/backend/plugin/src/test/resources/config/application.yml
Comment (laurens-ritense) [reply slack/backend/plugin/src/test/resources/config/application.yml:1]: I don't see any

Comment (Klaas-Ritense) [reply slack/backend/plugin/src/test/resources/config/application.yml:1]: Its here: Plugins/slack/backend/plugin/src/test/kotlin/com/ritense/valtimo/slack/plugin/SlackPluginIT.kt

**Review by Klaas-Ritense — APPROVED** [review]

**Pull request merged**